bin/sandbox

```diff
@@ -15,6 +15,7 @@ function unbundled {
 
 cd "./$app_name"
 unbundled bundle add $extension_name --path ..
+unbundled bundle add stripe-ruby-mock --group "test"
```

Comment:
Can you help me remember why we need this in the sandbox? I though having the dependency. in the dummy-app would be enough, but maybe I'm missing something.

Reply:
I wasn't sure if sandbox also suppose to include the new dependency - in such case I will remove it from `sandbox` and will leave it in `dummy-app`, thanks!